Sources/Solita/RenderInstruction.swift

```diff
@@ -54,6 +54,18 @@ public class InstructionRenderer {
         let swiftType = self.typeMapper.map(ty: arg.type, name: arg.name)
         return "\(arg.name): \(swiftType)"
     }
+
+    private func renderIxParametersField(arg: IdlField) -> String {
+        let swiftType = self.typeMapper.map(ty: arg.type, name: arg.name)
+        if arg.type.key.hasSuffix("?") {
```

Comment:
Don't love this, I started going pretty deep into the weeds with the `IdlType`, the time it was taking for an elegant solution wasn't worth it. @ajamaica you're more familiar with this code, so maybe I was missing a quick way to parse if the type is optional.